Sources/FileSystem/Configuration/TestflightConfigurationProcessor.swift

```diff
@@ -54,4 +58,43 @@ struct TestflightConfigurationProcessor {
         }
     }
 
+    func readConfiguration() throws -> TestflightConfiguration {
+        let folder = try Folder(path: path)
+        var configuration = TestflightConfiguration()
+
+        let decodeBetaTesters: (Data) throws -> [BetaTester] = { data in
```

Comment:
Just a suggestion:
```suggestion
extension Array where Element == BetaTester {
    init(data: Data) throws {
    }
}
```

Reply:
I believe this makes the code a lot less discoverable. I realise it's a common thing in this codebase, however I don't believe it's that common a practice. I also think there is merit in limiting the scope of functions, as in why should we expose this functionality more widely than the single place it is used?